src/main/scala/beam/utils/plan/PlansBuilder.scala

```diff
@@ -192,14 +192,12 @@ object PlansBuilder {
         val newPlan = PopulationUtils.createPlan(newPerson)
         newPerson.addPlan(newPlan)
         PopulationUtils.copyFromTo(pop(idx % pop.size).getPlans.get(0), newPlan)
+        val homeActs = newPlan.getPlanElements.asScala.filter(_.isInstanceOf[Activity]).map(pe => pe.asInstanceOf[Activity]).
```

Comment:
This can be simplified (and save on allocation (I believe)):

    val homeActs = newPlan.getPlanElements.asScala
      .collect{ case activity: Activity if activity.getType.equalsIgnoreCase("Home") => activity }

*Same for below